src/server/game/Spells/SpellMgr.cpp

```diff
@@ -7177,32 +7177,32 @@ void SpellMgr::LoadDbcDataCorrections()
         spellInfo->AttributesEx7 |= SPELL_ATTR7_INTERRUPT_ONLY_NONPLAYER;
     });
 
-    for (uint32 i = 0; i < sSpellStore.GetNumRows(); ++i)
+    for (uint32 i = 0; i < GetSpellInfoStoreSize(); ++i)
     {
-        SpellEntry* spellInfo = (SpellEntry*)sSpellStore.LookupEntry(i);
+        SpellInfo* spellInfo = mSpellInfoMap[i];
```

Comment:
if (and only if) `mSpellInfoMap[i]` is already a `SpellInfo*`, then we can use `auto` instead of `SpellInfo`

Reply:
so "auto* spellinfo"?

Reply:
auto& spellinfo

Reply:
Thanks Viste <3